tools/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -217,6 +218,30 @@ class QualificationAppInfo(
       val (allComplexTypes, nestedComplexTypes) = reportComplexTypes
       val problems = getAllPotentialProblems(getPotentialProblemsForDf, nestedComplexTypes)
 
+      val planInfos = sqlPlans.map { case (id, plan) =>
```

Comment:
The purpose of this variable for now is to log the total duration. Is it okay to assume `planInfos` would be used in qualification output later?